src/base/mouse/mouse.c

```diff
@@ -1755,6 +1778,11 @@ static void int33_mouse_move_buttons(int lbutton, int mbutton, int rbutton, void
 	   mouse_rb();
 }
 
+static void int33_mouse_move_wheel(int dy, void *udata)
+{
+	mouse.wmcount = dy;
```

Comment:
I think this whole function needs more stuff, but I'm not really sure yet what should be there. I think `mouse` is the dosemu struct and that function needs to pass the data to the dos environment itself.